host/sampi/http.go

```diff
@@ -53,15 +60,20 @@ func (s *Cluster) AddJobs(req map[string][]*host.Job) (map[string]host.Host, err
 }
 
 // Host Service methods
-
 func (s *Cluster) RemoveJobs(hostID string, jobIDs ...string) error {
+	l := s.logger.New("fn", "RemoveJobs", "host.id", hostID)
+	l.Debug("beginning to remove jobs")
 	s.state.Begin()
+	s.logger.Debug("now removing jobs from host")
 	s.state.RemoveJobs(hostID, jobIDs...)
+	s.logger.Debug("commiting")
```

Comment:
I think `Commit` should begin by logging the fact that a commit is about to happen, rather than each caller logging that fact.